### DISCUSSION TITLE
Lessons

### DISCUSSION THREAD

**Discussion by nathancoooper** [kantord/LibreLingo#2191] [Ideas]
It would be good if there was an option to display a page of explanation before teaching a term. For example, say someone is learning how to say enchante in the French course, it would be nice if there was an option to display a line of text stating when to use enchante and when to use enchantee (masc. and fem.)

**Comment by cutthroat78** [comment]
If you place a markdown file with the same names as the skill, the markdown file will be shown before starting the skill. [Docs](https://github.com/LibreLingo/LibreLingo/blob/main/docs/courses/skill.md#markdown) I know this is different to what you say but another idea you could do is to include notes like that in the mini-dictionary and so it will be displayed when you hover over the term

**Reply by nathancoooper** [reply]
This is perfect, thank you!

**Reply by kantord** [reply]
The idea of being able to add notes to the mini dictionary is also interesting
